### ISSUE TITLE
The app crashes and leads me to a line of code in the SideMenu pod

### ISSUE THREAD

**Issue by ahmedbahgaten** [jonkykong/SideMenu#625]
My problem is that I log into my app and the side menu works just fine but when I log out of the app and re-login, there is an annoying crash that leads me to a line of code in the pod file. the crash only happens when I log out of the app and re-log again
the issue related to memory management as there is an object the code tries to access and it's not accessible anymore.  
 
![Screen Shot 2020-04-24 at 1 53 20 AM](https://user-images.githubusercontent.com/57404394/80160694-84715b00-85ce-11ea-8b67-643b0c654d7d.png)

**Comment by wolfej94** [comment]
I've been struggling with this one for a while, I'm assuming you have more than one instance of a side menu in your app?

**Comment by ahmedbahgaten** [comment]
> I've been struggling with this one for a while, I'm assuming you have more than one instance of a side menu in your app?

Yes, I do have more than an instance of the side menu in my app

**Comment by wolfej94** [comment]
https://github.com/jonkykong/SideMenu/issues/435

**Comment by wolfej94** [comment]
What I have done to fix this is to move the side menu code into the app delegate and trigger it's display by posting via the NotificationCenter that trigger selectors in the app delegate. 

```
    var sideMenu: SideMenuNavigationController?

    @objc func showDriverMenu() {
        let vc = UIStoryboard(name: "Main", bundle: .main).instantiateViewController(withIdentifier: "DriverMenu")
        showMenu(vc, side: .left)
    }

    public func showMenu(_ vc: UIViewController, side: SideMenuPresentationSide) {
        guard let rootVC = window?.rootViewController else { return }
        sideMenu = SideMenuNavigationController(rootViewController: vc)
        sideMenu!.statusBarEndAlpha = 0
        sideMenu!.menuWidth = rootVC.view.frame.width * 0.75
        sideMenu!.leftSide = side == .left
        
        rootVC.present(sideMenu!, animated: true, completion: nil)
    }
```

Make sure if you're setting the rootView controller of the window to dismiss your current instance of SideMenu before doing so!

**Comment by ahmedbahgaten** [comment]
Thank you so much, you saved many days that might have been wasted if you didn't help me.

**Comment by jonkykong** [comment]
Fixed in `6.4.9`.

**Comment by RobertHaworth** [comment]
> Fixed in `6.4.9`.

Can we get a tag for 6.4.9 so that SPM will pull in this bugfix?

**Comment by jonkykong** [comment]
Done.
